sdk/keyvault/keyvault-certificates/CHANGELOG.md

```diff
@@ -1,15 +1,12 @@
 # Release History
 
-## 4.3.0-beta.1 (Unreleased)
+## 4.3.0 (2021-07-29)
 
 ### Features Added
-- With the dropping of support for Node.js versions that are no longer in LTS, the dependency on `@types/node` has been updated to version 12. Read our [support policy](https://github.com/Azure/azure-sdk-for-js/blob/main/SUPPORT.md) for more details.
-
-### Breaking Changes
 
-### Key Bugs Fixed
-
-### Fixed
+- With the dropping of support for Node.js versions that are no longer in LTS, the dependency on `@types/node` has been updated to version 12. Read our [support policy](https://github.com/Azure/azure-sdk-for-js/blob/main/SUPPORT.md) for more details.
+- Changed TS compilation target to ES2017 in order to produce smaller bundles and use more native platform features
```

Comment:
This line is missing a dot at the end, and I wonder if a link where would help? for further context? Same in the other changelog

Reply:
```suggestion
- Changed TS compilation target to ES2017 in order to produce smaller bundles and use more native platform features.
```